selfdrive/car/ford/values.py

```diff
@@ -44,11 +44,12 @@ class CAR:
   BRONCO_SPORT_MK1 = "FORD BRONCO SPORT 1ST GEN"
   ESCAPE_MK4 = "FORD ESCAPE 4TH GEN"
   EXPLORER_MK6 = "FORD EXPLORER 6TH GEN"
+  F_150_MK14 = "FORD F-150 14TH GEN"
   FOCUS_MK4 = "FORD FOCUS 4TH GEN"
   MAVERICK_MK1 = "FORD MAVERICK 1ST GEN"
 
 
-CANFD_CARS: Set[str] = set()
+CANFD_CARS = {CAR.F_150_MK14}
```

Comment:
can we rename to `CANFD_CAR`?

Reply:
e7c858d590f84c4f476ec0a01eae67f49f7a114b